copter/source/docs/setting-up-for-tuning.rst

```diff
@@ -25,8 +25,8 @@ There are 3 common causes of a non-linear thrust curve.
 Start with setting up the voltage range to cope with voltage sag.
 Parameters used to linearise your motor thrust curve.
 
-- :ref:`MOT_BAT_VOLT_MAX <MOT_BAT_VOLT_MAX>`: 4.2v x No. Cells
-- :ref:`MOT_BAT_VOLT_MIN <MOT_BAT_VOLT_MIN>`: 3.3v x No. Cells
+- :ref:`MOT_BAT_VOLT_MAX <MOT_BAT_VOLT_MAX>` should be set to the maximum voltage of your batteries, e.g. 4.2v x No. Cells for standard LiPos.
```

Comment:
```suggestion
- :ref:`MOT_BAT_VOLT_MAX <MOT_BAT_VOLT_MAX>` should be set to the maximum voltage of your batteries, e.g. 4.2v x No. Cells for standard LiPos, or as appropriate if using a different battery type .
```

Reply:
Well, putting the e.g. was meant to mean the 4.2v is just an example.  I've already got "of your batteries" so I'm not sure we need the extra bit as well?
Or should we change the "e.g." to "i.e.", so it makes at least a bit more sense?

Reply:
I *do* know the difference between eg and ie, actually...
In fact, I was thinking *precisely* "that is" would fit better than "for example".

If we're giving just one example - like just mentioning LiPos and nothing else as my original PR had, then "eg" would make sense.
But "4.2v x No. Cells for standard LiPos, or as appropriate if using a different battery type" is effectively all possible options, therefore "for example" doesn't make sense anymore and "that is" makes more sense.